WeakAuras/Prototypes.lua

```diff
@@ -5539,9 +5595,12 @@ Private.event_prototypes = {
       {
         name = "spellId",
         display = L["Spell Id"],
-        type = "string",
+        type = "spell",
```

Comment:
This changes the type from a string to a number, is that backwards compatible? Do existing auras continue to work?

Reply:
no it's still a string
![image](https://github.com/WeakAuras/WeakAuras2/assets/189656/f09aaa89-bb61-4797-be0c-15c36511ec15)

Reply:
Oh right, then that should be ok.